extensions/amp-story-panning-media/amp-story-panning-media.md

```diff
@@ -0,0 +1,159 @@
+---
+$category@: media
+formats:
+  - stories
+teaser:
+  text: >-
+    Transition an image's position and zoom between pages.
+draft:
+  - true
+tags:
+  - immersive
+  - panning
+  - zooming
+  - animation
+  - media
+  - transitions
+author: processprocess
+toc: true
+$title: amp-story-panning-media
+version: '0.1'
+versions:
+  - '0.1'
+latest_version: '0.1'
+is_current: true
+$path: /documentation/components/amp-story-panning-media.html
+$localization:
+  path: '/{locale}/documentation/components/amp-story-panning-media.html'
+scripts:
+  - >-
+    <script async custom-element="amp-story-panning-media"
+    src="https://cdn.ampproject.org/v0/amp-story-panning-media-0.1.js"></script>
+layouts:
+  - Fill
+---
+
+<!---
+Copyright 2021 The AMP HTML Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS-IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+<figure class="centered-fig">
+  <amp-anim alt="amp-story-panning-media example" width="304" height="548" layout="fixed" src="https://raw.githubusercontent.com/processprocess/amphtml/panning-media-docs/extensions/amp-story-panning-media/img/amp-story-panning-media.gif">
+    <noscript>
+    <img alt="amp-story-panning-media example" src="https://raw.githubusercontent.com/processprocess/amphtml/panning-media-docs/extensions/amp-story-panning-media/img/amp-story-panning-media.gif" />
+  </noscript>
+  </amp-anim>
+</figure>
+
+The `amp-story-panning-media` component provides a way to pan and zoom an image between pages in [Web Stories](https://amp.dev/documentation/guides-and-tutorials/start/create_successful_stories/?format=stories).
+
+## Usage
+
+Use the `amp-story-panning-media` to transition an image between pages. The component can animate between position and zoom.
+
+## Environment setup
+
+Add the following import to your AMP documents:
+
+```html
+<script async custom-element="amp-story-panning-media" src="https://cdn.ampproject.org/v0/amp-story-panning-media-0.1.js"><script>
+```
+
+### Configuration
+
+The `amp-story-panning-media` component accepts 1 descendant. This descendant must be an [amp-img](https://amp.dev/documentation/components/amp-img/?format=stories). Components associate with each other when they have a child with the same `src`.
```

Comment:
Could you explain "Components associate with each other when they have a child with the same src" further? I'm not sure what this means and what I should be doing as a developer.